reference/forms/types/options/choice_value.rst.inc

```diff
@@ -12,7 +12,10 @@ about this, but it might be handy when processing an API request (since you can
 configure the value that will be sent in the API request).
 
 This can be a callable or a property path. See `choice_label`_ for similar usage.
-If ``null`` is used, an incrementing integer is used as the name.
+*If you use a callable for this option, it gets passed the choice as its **only**
+argument*. This is a difference compared to the callables of `choice_label`_ and
+`choice_name`_, which receive additional arguments. If ``null`` is used, an
+incrementing integer is used as the name.
```

Comment:
:+1: 

This is a detail but I would say :
```rst
.. note::

    In contrast to other choice options using a callable (e.g ``choice_label``),
    the ``choice_value`` option takes as **only** argument the actual choice.
```
Because the actual enumeration misses `choice_attr` (and maybe future [other(s)](symfony/symfony#16834) ;).

Anyway please remove ``See `choice_label`_ for similar usage.``, as I really think it can confuse new comers.